new-content/learning-resources/qiskit-blog.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: Qiskit blog
-to: https://medium.com/qiskit
-image: /images/landing-page/feature-hardware.jpg
-ctaLabel: Read the docs
+title: Qiskit Medium
+to: https://medium.com/qiskit/qiskit-and-its-fundamental-elements-bcd7ead80492
```

Comment:
This link point to a post related to qiskit elements. As far as I know, qiskit elements will disappear, so, what about pointing to the main qiskit page on medium?

Reply:
The elements are still a part of the qiskit SDK so when you learn how to use Qiskit you will still run into them. In this case - it's ok to use them.